### DISCUSSION TITLE
is Pixi v6 production ready?

### DISCUSSION THREAD

**Discussion by goldenratio** [pixijs/pixijs#8238] [Q&A] [answered]
We are on pixi v5, planning to update to v6.
is Pixi v6 production ready?

**Comment by bigtimebuddy** [comment, marked as answer]
Yes it is. Many folks have been using v6 in production for awhile now. The upgrade should be relatively smooth since v5 and v6 are about the same, but with better typings. Please read the [v6.0.0 release](https://github.com/pixijs/pixijs/releases/tag/v6.0.0) about breaking changes.

**Reply by goldenratio** [reply]
thank you
